digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/masstransitstop/oracle/queries.scala

```diff
@@ -237,6 +237,13 @@ object Queries {
     """.as[Int].list
   }
 
+  def getMunicipalitiesWithoutAhvenanmaa: Seq[Int] = {
+    var AhvenanmaaRoadMaintainerId = 0
```

Comment:
I'm going even deepely than that, if the whole purpose of the variable AhvenanmaaRoadMaintainerId is to be referenced in the SQL with _"#$AhvenanmaaRoadMaintainerId"_ and never to be updated then why isn't this scraped and the SQL switched to _"select id from municipality where **ROAD_MAINTAINER_ID != 0**"_